Formula/b/bootterm.rb

```diff
@@ -0,0 +1,19 @@
+class Bootterm < Formula
+  desc "Simple, reliable and powerful terminal to ease connection to serial ports"
+  homepage "https://github.com/wtarreau/bootterm"
+  url "https://github.com/wtarreau/bootterm/archive/refs/tags/v0.5.tar.gz"
+  sha256 "95cc154236655082fb60e8cdae15823e4624e108b8aead59498ac8f2263295ad"
+  license "MIT"
+  head "https://github.com/wtarreau/bootterm.git", branch: "master"
+
+  depends_on xcode: :build
```

Comment:
I'm not entirely sure you need this. Could you try without this dependency?

Reply:
Since this is a C project, I thought xcode is required to provide a c compiler in order to build the project.